### PR TITLE
Subscriptions

### PR DESCRIPTION
**Pull request by cap10morgan**
This adds the basic Datomic plumbing to support subscriptions. The remaining work will be in additional cards.

[Pivotal card](https://www.pivotaltracker.com/story/show/87670334)

Assigning to @bwreid.

### CODE REVIEW

File: src/ballot_notification_works/core.clj [hunk 1]
```diff
@@ -18,7 +19,7 @@
     (log/info "Handling a ballot status update message")
     (log/debug message)
     (let [subject (str "Your new ballot status is "
-                       (-> message :new-status :status name))]
+                       (-> message :ballot-after :new-status :status name))]
```
Comment (bwreid) [inline src/ballot_notification_works/core.clj]: Does this relate to this card or is this just happening?

**Comment by bwreid** [comment]
Looks like this needs a master merge.

File: src/ballot_notification_works/core.clj (hunk 1 above)
Comment (cap10morgan) [inline src/ballot_notification_works/core.clj]: It's an unrelated bug fix that went into this branch b/c I found a few seconds before I demo'd this. Seemed tiny enough to leave in.

**Comment by bwreid** [comment]
el gee tee em 2 me.
